src/components/Home/styles.module.scss

```
@@ -4,9 +4,82 @@
 }
 .hr {
     display: block;
-    margin-top: 5px;
-    margin-bottom: 5px;
+    margin-top: 8px;
+    margin-bottom: 8px;
     background-color: gray;
     opacity: 0.3;
     height: 2px;
-}
\ No newline at end of file
+}
+
+.cont1 {
+    width:100%;
```

Presta atención al espaciado entre propiedades y valores. En la [guía](https://docs.google.com/document/d/1gJ0gB1NkinQXyRe5dk71yMUYE8c6dupLc-lYQ0osSro) vas a encontrar más información sobre por qué es importante. 
```suggestion
    width:100%;
```